### PR TITLE
Fix Typos in 7 Unexpected Ways Verifiable Credentials Are Used Today Blog Post

### LINKED ISSUE #1572: Fix Typos in 7 Unexpected Ways Verifiable Credentials Are Used Today Blog Post
# Fix Typos in 7 Unexpected Ways Verifiable Credentials Are Used Today Blog Post

## 🚀 Goal
We want to ensure our website's blog posts don't have any mistakes.

## 🤔 Background
This issue focuses specifically on fixing typos across on one of our blogs. Your contributions here make a difference!

## 🤝 How You Can Contribute
We've broken it down into specific typos for you to address. Every contribution counts!

### **❗You must only assign yourself one task at a time to give everyone a chance to participate.❗**
You may assign yourself your next task after your current task is reviewed & accepted.
1. Fix typos in 7 Unexpected Ways Verifiable Credentials Are Used Today Blog Post:
-  [ ] Change "Verifiable Credentials seem niche." to "Verifiable Credentials seems niche." to represent the singular concept
image
2. Make sure all tests are passing.

## 🌟 Resources
- [7 Unexpected Ways Verifiable Credentials Are Used Today Blog Post](https://developer.tbd.website/blog/7-ways-we-use-vcs-today)
- [Blog, Developer Website](https://developer.tbd.website/blog)

## Getting Started
1. Comment ".take" on this issue to get assigned
2. Fork the repository and create a new branch for this task
4. Follow the tasks outlined above
5. Submit a pull request with your changes
6. Respond to any feedback during the review process

## Questions?
If you have any questions or need clarification, please comment on this issue or join our [Discord](https://discord.gg/tbd) community.

Happy coding! 🎉

### PR DESCRIPTION
**Pull request by Nabhel**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ New Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 👷 Example Application
- [ ] 🧑‍💻 Code Snippet
- [ ] 🎨 Design
- [ ] 📖 Content
- [ ] 🧪 Tests
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

This PR fixes typos in 7 Unexpected Ways Verifiable Credentials Are Used Today Blog Post. 

## Related Tickets & Documents

Resolves #1572

## Added code snippets?
- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 📓 general documentation
- [x] 🙅 no documentation needed

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -20,10 +20,10 @@ jobs:
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
```
Comment (blackgirlbytes) [inline .github/workflows/main.yml:23]: Hi @Nabhel . Can you please revert the changes in the main.yaml GitHub action as this is out of scope for your task?

**Review by blackgirlbytes — CHANGES_REQUESTED** [review]

**Comment by blackgirlbytes** [comment]
Hi @Nabhel can you please also fill out the PR description with the issue that this resolves and what type of fix it is? This will convert those failed GitHub Actions into passing tests. Thank you!

File: .github/workflows/main.yml (hunk 1 above)
Comment (Nabhel) [reply .github/workflows/main.yml]: Hi @blackgirlbytes I have reverted the changes. Thanks

**Review by blackgirlbytes — APPROVED** [review]
Thank you for your contribution!
